gen/dcos-config.yaml

```diff
@@ -1125,3 +1117,100 @@ package:
+        collection_jitter = "0s"
+        ## Default flushing interval for all outputs. You shouldn't set this below
+        ## interval. Maximum flush_interval will be flush_interval + flush_jitter
+        flush_interval = "10s"
+        ## Jitter the flush interval by a random amount. This is primarily to avoid
+        ## large write spikes for users running a large number of telegraf instances.
+        ## ie, a jitter of 5s and interval 10s means flushes will happen every 10-15s
+        flush_jitter = "0s"
+        ## By default or when set to "0s", precision will be set to the same
+        ## timestamp order as the collection interval, with the maximum being 1s.
+        ##   ie, when interval = "10s", precision will be "1s"
+        ##       when interval = "250ms", precision will be "1ms"
+        ## Precision will NOT be used for service inputs. It is up to each individual
+        ## service input to set the timestamp at the appropriate precision.
+        ## Valid time units are "ns", "us" (or "µs"), "ms", "s".
+        precision = ""
+        ## Logging configuration:
+        ## Run telegraf with debug log messages.
+        debug = false
+        ## Run telegraf in quiet mode (error log messages only).
+        quiet = false
+        ## Specify the log file name. The empty string means to log to stderr.
+        logfile = ""
+        ## Override default hostname, if empty use os.Hostname()
+        hostname = ""
+        ## If set to true, do no set the "host" tag in the telegraf agent.
+        omit_hostname = false
+      # Publish all metrics to /metrics for Prometheus to scrape
+      [[outputs.prometheus_client]]
+        # Address to listen on
+        listen = ":61091"
+        # Path to publish the metrics on, defaults to /metrics
+        path = "/metrics"
+        # Expiration interval for each metric. 0 == no expiration
+        expiration_interval = "60s"
+        # Send string metrics as Prometheus labels.
+        # Unless set to false all string metrics will be sent as labels.
+        string_as_label = true
+      # Read metrics about cpu usage
+      [[inputs.cpu]]
+        ## Whether to report per-cpu stats or not
+        percpu = true
+        ## Whether to report total system cpu stats or not
+        totalcpu = true
+        ## If true, collect raw CPU time metrics.
+        collect_cpu_time = false
+        ## If true, compute and report the sum of all non-idle CPU states.
+        report_active = false
+      [[inputs.mem]]
+  - path: /etc/_master/telegraf/telegraf.d/master.conf
+    content: |
+      # Additional Telegraf config for masters
+      # TODO(branden)
+  - path: /etc_slave/telegraf/telegraf.d/agent.conf
+    content: |
+      # Additional Telegraf config for agents
+      # Telegraf plugin for gathering resource metrics about mesos containers
+      [[inputs.dcos_containers]]
+        ## The address of the mesos agent
+        agent_address = "http://localhost:5051"
```

Comment:
should this be called `mesos_agent_address`?

Reply:
That sgtm, that's something we'd need to fix upstream. /cc @philipnrmn

Reply:
You can make this change immediately, I'll fix it upstream now.

Reply:
let's go with 'mesos_agent_url', it's neater.